src/components/expression/expression.js

```diff
@@ -79,11 +86,13 @@ genotet.ExpressionView = function(viewName, params) {
 
   // Set up data loading callbacks.
   $(this.container).on('genotet.ready', function() {
+    this.data.tfa.fileName = params.tfaFileName;
```

Comment:
ditto. I think matrix should not be necessarily chained after tfa. Expression should be able to support matrix without TFA. Sorry that I didn't realize this mistake when I reviewed the previous pullreq.